backend/src/core/middlewares/file-attachment.middleware.ts

```diff
@@ -58,6 +59,10 @@ async function checkAttachmentValidity(
   _res: Response,
   next: NextFunction
 ): Promise<void> {
+  if (!req.files?.attachments && req.body.attachments) {
```

Comment:
If both `req.files?.attachments` and `req.body.attachments` are defined, then we are still ok to continue running this function?

Reply:
Hmm good point. In practice, that should never happen because `req.files?.attachments` can only be defined if they use multi-part upload and `req.body.attachments` can. only be defined if they're using JSON. 

But a blanket ban on all JSON `attachments` is more consistent with what our `openapi.yaml` says. Let me change it then! Thanks!

Reply:
Hmm no this doesn't play well with [the tests](https://github.com/opengovsg/postmangovsg/actions/runs/5863380143/job/15896728265?pr=2179), however. Let me revert the change!

Reply:
Thank you, Zi Xiang!

Reply:
haha you've merged alr! looking at the code now, I think it's clearer if the validation is done here: 
![Uploading image.png…]()

but no matter!